- Biorad planning ddPCR experiments guide: https://www.bio-rad.com/en-us/life-science/learning-center/introduction-to-digital-pcr/planning-ddpcr-experiments 
- TaqMan Assay Multiplex PCR Optimization Guide: http://assets.thermofisher.com/TFS-Assets/LSG/manuals/taqman_optimization_man.pdf

Multiplex primer design tips:
1. The primer melting temperature (Tm) should be the same for all primers used in the multiplex
reaction.
2. Plan to amplify a 75–200 bp product. Short PCR products are typically amplified with higher efficiency than longer ones, but a PCR product should be at least 75 bp long to allow its discrimination from any primer-dimers that may form
3. Ensure that primers do not contain bases that are complementary to other bases within the primer
(self-complementary), or have complementarity to other primers. Complementarity at the 3' ends
should be avoided to minimize the formation of artifact products, for example, primer-dimers or
primer-oligomers
4. Ensure that the last five nucleotides at the 3´ end contain no more than two G and/or C bases.
5. Avoid regions that have secondary structure when possible. Use programs such as mfold to predict whether primers will anneal to form a secondary structure or a PCR product will form any significant secondary structure at the annealing temperature

General primer design tips
1. Aim for the GC content to be between 40 and 60% with the 3’ of a primer ending in G or C to promote binding. This is known as a GC Clamp. The G and C bases have stronger hydrogen bonding and help with the stability of the primer. Be mindful not to have too many repeating G or C bases, as this can cause primer-dimer formation.
2. A good length for PCR primers is generally around 18-30 bases. Specificity usually is dependent on length and annealing temperature. The shorter the primers are, the more efficiently they will bind or anneal to the target.
3. Try to make the melting temperature (Tm) of the primers between 58°C and 60°C, and within 1°C of each other. Because the Tm is dependent on the length, it’s important to keep primers on the shorter end. The bases also impact the Tm, G and C result in higher melting temperatures than A and T. If the Tm of your primer is very low, try to find a sequence with more GC content, or extend the length of the primer a little.
4. Typically, 3 to 4 nucleotides are added 5 ’of the restriction enzyme site in the primer to allow for efficient cutting.
5. Try to avoid regions of secondary structure and have a balanced distribution of GC-rich and AT-rich domains.
6. Try to avoid runs of 4 or more of one base, or dinucleotide repeats (for example, ACCCC or ATATATAT).
7 Avoid intra-primer homology (more than 3 bases that complement within the primer) or inter-primer homology (forward and reverse primers having complementary sequences). These circumstances can lead to self-dimers or primer-dimers instead of annealing to the desired DNA sequences.

General probe design tips
1. The Tm of each hydrolysis probe should be 5–10°C higher than that of the corresponding primers. In most cases, the probe should have <30 nucleotides between the fluorophore and the quencher to avoid affecting baseline signal intensity. It must not have a G at its 5' end, because this may quench the fluorescence signal even after hydrolysis. Choose a sequence within the target that has a GC content of 30–80% and design the probe to anneal to the strand that has more Gs than Cs (so the probe contains more Cs than Gs).
2. Black Hole quenchers are recommended
