TaqMan Assay Multiplex PCR Optimization Guide: http://assets.thermofisher.com/TFS-Assets/LSG/manuals/taqman_optimization_man.pdf

Multiplex primer design tips:
1. The primer melting temperature (Tm) should be the same for all primers used in the multiplex
reaction.
2. The length of the amplicon should be 50–150 bp for optimal PCR efficiency. If longer amplicons
cannot be avoided, it may be necessary to optimize the thermal cycling protocol and reaction
components
3. Ensure that primers do not contain bases that are complementary to other bases within the primer
(self-complementary), or have complementarity to other primers. Complementarity at the 3' ends
should be avoided to minimize the formation of artifact products, for example, primer-dimers or
primer-oligomers
4. Ensure that the last five nucleotides at the 3´ end contain no more than two G and/or C bases.

General primer design tips
1. Aim for the GC content to be between 40 and 60% with the 3’ of a primer ending in G or C to promote binding. This is known as a GC Clamp. The G and C bases have stronger hydrogen bonding and help with the stability of the primer. Be mindful not to have too many repeating G or C bases, as this can cause primer-dimer formation.
2. A good length for PCR primers is generally around 18-30 bases. Specificity usually is dependent on length and annealing temperature. The shorter the primers are, the more efficiently they will bind or anneal to the target.
3. Try to make the melting temperature (Tm) of the primers between 65°C and 75°C, and within 5°C of each other. Because the Tm is dependent on the length, it’s important to keep primers on the shorter end. The bases also impact the Tm, G and C result in higher melting temperatures than A and T. If the Tm of your primer is very low, try to find a sequence with more GC content, or extend the length of the primer a little.
4. Typically, 3 to 4 nucleotides are added 5 ’of the restriction enzyme site in the primer to allow for efficient cutting.
5. Try to avoid regions of secondary structure and have a balanced distribution of GC-rich and AT-rich domains.
6. Try to avoid runs of 4 or more of one base, or dinucleotide repeats (for example, ACCCC or ATATATAT).
7 Avoid intra-primer homology (more than 3 bases that complement within the primer) or inter-primer homology (forward and reverse primers having complementary sequences). These circumstances can lead to self-dimers or primer-dimers instead of annealing to the desired DNA sequences.
